docs/tutorials/NMR/Example_1_-Ethanol.md

```diff
@@ -0,0 +1,115 @@
+<img alt="Fig1. Proton spectrum of ethanol" src="../../img/nmr_tut1.png" width = "300"/>
+<figure fig1>
+  <figcaption>Fig1. Proton spectrum of ethanol</figcaption>
+</figure>
```

Comment:
It should be noted that mkdocs also supports [extended markdown attributes](https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions):


```suggestion
<figure markdown="span">
  ![Fig. 1 Proton spectrum of ethanol](../../img/nmr_tut1.png){ width="300" }
  <figcaption>Fig. 1 Proton spectrum of ethanol</figcaption>
</figure>
```

Reply:
It should be noted, by the way, that `nmr_tut1.png` hasn't been added in this PR.

Reply:
I tried puting the image inside <figure markdown="span"> (I figured that might solve the issue of there being an awkward space between the figure and the figure caption) but it didn't work - it just gave out the "![Fig.1 ..." in text form. 

nmr_tut1.png wasn't added in the pull request because it was already there (it was the only image I didn't have to add), but I got the source wrong - weirdly it can still find the image even when you do that when you add too many "..." in the source